packages/react-inbox/src/hooks/user-at-bottom.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  useCallback, useEffect, useState,
+} from "react";
+
+export function useAtBottom(element) {
```

Comment:
oh, decided to use something different from the libraries we shared earlier?

Reply:
Yes, little bit more lightweight and flexible